src/pyOpenMS/tests/unittests/test_miscellaneous.py

```
@@ -0,0 +1,809 @@
+
+    m.clear()
+    m.clearUniqueId()
+    m.ensureUniqueId()
+    m.getDataProcessing()
+    m.getColumnHeaders()
+    m.getProteinIdentifications()
+    m.getUnassignedPeptideIdentifications()
+    m.getUniqueId()
+    m.hasInvalidUniqueId()
+    m.hasValidUniqueId()
+    m.setDataProcessing
```

Ruff (0.14.10)

Found useless expression. Either assign it to a variable or remove it.

(B018)